helix-view/src/view.rs

```diff
@@ -91,12 +95,23 @@ impl fmt::Debug for View {
 impl View {
     pub fn new(doc: DocumentId, gutter_types: Vec<crate::editor::GutterType>) -> Self {
         let mut gutters: Vec<(Gutter, usize)> = vec![];
+        let mut gutter_offset = 0;
         use crate::editor::GutterType;
         for gutter_type in &gutter_types {
-            match gutter_type {
-                GutterType::Diagnostics => gutters.push((gutter::diagnostics_or_breakpoints, 1)),
-                GutterType::LineNumbers => gutters.push((gutter::line_numbers, 5)),
-            }
+            let width = match gutter_type {
+                GutterType::Diagnostics => 1,
+                GutterType::LineNumbers => 5,
+                GutterType::Padding => 1,
+            };
+            gutter_offset += width;
+            gutters.push((
+                match gutter_type {
+                    GutterType::Diagnostics => gutter::diagnostics_or_breakpoints,
+                    GutterType::LineNumbers => gutter::line_numbers,
+                    GutterType::Padding => gutter::padding,
+                },
```

Comment:
Not sure why but I can't seemed to keep these as a variable `gutter`, maybe function name is different?